pkgs/desktops/gnome-3/core/gnome-settings-daemon/default.nix

```diff
@@ -93,6 +94,12 @@ stdenv.mkDerivation rec {
     "-Dudev_dir=${placeholder "out"}/lib/udev"
   ];
 
+  # So the polkit policy can reference /run/current-system/sw/bin/gsd-backlight-helper
+  postFixup = ''
+    mkdir $out/bin
+    ln -s $out/libexec/gsd-backlight-helper $out/bin/gsd-backlight-helper
```

Comment:
Can we use some other path that is guaranteed to be linked but not in the path?

Reply:
Maybe a subdirectory of `/bin`

Reply:
Doing `bin/gnome-settings-daemon`.

Reply:
Could `pathsToLink = [ "/libexec/gsd-backlight-helper" ];` `gnome-settings-daemon.nix` work? Seems somewhat cleaner to me, though not a big deal.

Reply:
Yeah thought of that too. It'd be cleaner but I don't see it happening soon where `gnome-settings-daemon` wouldn't be in `environment.systemPackages`. (thusly needing that).